src/main/kotlin/com/example/mykku/feed/domain/FeedComment.kt

```
@@ -0,0 +1,25 @@
+    @Column(name = "like_count")
+    var likeCount: Int = 0,
+
+    @ManyToOne
+    @JoinColumn(name = "feed_id")
+    val feed: Feed,
+
+    @ManyToOne
+    @JoinColumn(name = "parent_comment_id")
+    val parentComment: FeedComment? = null,
+) {
+}
```

detekt (1.23.8)

[warning] The class or object FeedComment is empty.

(detekt.empty-blocks.EmptyClassBlock)